frontend/src/utils.ts

```diff
@@ -59,3 +59,9 @@ export const removeContentWithMissingImagePath = (list: [], pathName: string) =>
 export const sortListByPopularity = (list: [any]) => {
   list.sort((a, b) => b.popularity - a.popularity)
 }
+
+export const handleSelect = (e, selectList) => {
+  if (e.detail) {
+    e.detail.forEach(({ value }) => selectList.push(value))
```

Comment:
This builds the list exponentially until all the elements are cleared
![image](https://user-images.githubusercontent.com/43907402/154859542-3fc5fa9e-23ee-4bbc-b43a-94feb8c42845.png)

Consider using
```suggestion
    selectList = e.detail.map(x => x.value)
```

Reply:
Damn bro, that's kind of embarassing. Fixed!